src/server.js

```diff
@@ -215,5 +224,91 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_ADDRESS}/geocode`,
+  async (request, response) => {
+    let address = request.params[URL_PARAM_ADDRESS];
+
+    let geocodeRequest =
+      "https://maps.googleapis.com/maps/api/geocode/json?address=" +
+      address +
+      "&key=" +
+      env.API_KEY;
+
```

Comment:
I suggest it would be better to specify which API key, as we may be using multiple API's. Something like API_KEY_GEOCODE. Also remember to put it in the env.sample

Reply:
Do you mean we may be using multiple api _keys?_. I was planning to change the name of the api key variable to whatever name you gave it in your pull request because we currently only use one api key for everything.

Reply:
Since all our API's are under GCP, we may end up needing only one key. But, if we used some external APIs, we would have needed multiple. That's where the need for multiple names come in. I specified my API key as API_KEY_PLACES, so we are not in sync. I feel in this situation, we could decide on either way and stick to it for consistency.

Reply:
I created a new environmental variable called API_KEY_GEOCODE.